### PR TITLE
Fix non-oak_debug builds

### PR DESCRIPTION
**Pull request by daviddrysdale**
# Checklist

- [x] Pull request affects core Oak functionality (e.g. runtime, SDK, ABI)
  - [x] I have written tests that cover the code changes.
  - [x] I have checked that these tests are run by
        [Cloudbuild](/cloudbuild.yaml)
  - [ ] I have updated [documentation](/docs/) accordingly.
  - [ ] I have raised an [issue](https://github.com/project-oak/oak/issues) to
        cover any TODOs and/or unfinished work.
- [ ] Pull request includes prototype/experimental work that is under
      construction.

### CODE REVIEW

**Comment by daviddrysdale** [comment]
(Deliberately incomplete fix for now to check that CI fails to build the `logless` / `--no-default-features` version of `oak_loader`)

**Comment by daviddrysdale** [comment]
For #1862

File: oak_runtime/src/lib.rs [hunk 1]
```diff
@@ -435,6 +435,7 @@ pub struct Runtime {
     aux_servers: Mutex<Vec<AuxServer>>,
 
     /// Queue of introspection events in chronological order.
+    #[allow(dead_code)]
```
Comment (rbehjati) [inline oak_runtime/src/lib.rs:438]: I wish this could have been avoided, but I cannot think of a better alternative.

**Review by rbehjati — APPROVED** [review]
Thanks!

**Pull request merged**